src/tests/ie_test_utils/common_test_utils/data_utils.cpp

```diff
@@ -180,7 +185,7 @@ void fill_data_with_broadcast(InferenceEngine::Blob::Ptr& blob, size_t axis, std
     fill_data_with_broadcast(blob, values_blob);
 }
 
-InferenceEngine::Blob::Ptr make_reshape_view(const InferenceEngine::Blob::Ptr &blob, InferenceEngine::SizeVector new_shape) {
+InferenceEngine::Blob::Ptr make_reshape_view(const InferenceEngine::Blob::Ptr &blob, SizeVector new_shape) {
```

Comment:
`InferenceEngine::Blob`???
Anyway, we need to review this file to understand what to do

Reply:
ok